src/Build/Graph/ProjectGraph.cs

```diff
@@ -599,6 +606,19 @@ internal string ToDot(Func<ProjectGraphNode, string> nodeIdProvider)
             }
 
             return targetLists;
+
+            void ThrowOnEmptyTargetNames(ICollection<string> targetNames)
+            {
+                if (targetNames == null || targetNames.Count == 0)
+                {
+                    return;
+                }
+
+                if (targetNames.Any(targetName => string.IsNullOrWhiteSpace(targetName)))
+                {
+                    throw new ArgumentException(ResourceUtilities.FormatResourceStringIgnoreCodeAndKeyword("OM_TargetNameNullOrEmpty"));
```

Comment:
Would it be possible to have more context here? Right now it just says there's a null/empty target name somewhere. How do I know where?